code/modules/reagents/chemistry/recipes/drinks.dm

```diff
@@ -876,3 +876,12 @@
 	required_reagents = list("ice" = 1, "hot_coco" = 3)
 	result_amount = 4
 	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/chemical_reaction/fernet_cola
+	name = "Fernet Cola"
+	id = "fernet_cola"
+	result = "fernet_cola"
+	required_reagents = list("fernet" = 1, "cola" = 2, "ice" = 1)
```

Comment:
I'd skip the ice. None of the other drinks that are usually made with ice or have ice in the sprites need it.

Reply:
Sure, I can do that